sklearn/neighbors/_regression.py

```diff
@@ -91,6 +91,8 @@ class KNeighborsRegressor(KNeighborsMixin, RegressorMixin, NeighborsBase):
         ``-1`` means using all processors. See :term:`Glossary <n_jobs>`
         for more details.
         Doesn't affect :meth:`fit` method.
+        Note that even if n_jobs=1, low-level parallelism (via Numpy and OpenMP)
+        might be used in some configuration. See :ref:`User Guide <computing>`.
```

Comment:
I think this statement is better suited for the `n_jobs` glossary entry because applies to all estimators that accept `n_jobs`.